akka-remote/src/test/scala/akka/remote/RemotingSpec.scala

```diff
@@ -818,6 +820,31 @@ class RemotingSpec extends AkkaSpec(RemotingSpec.cfg) with ImplicitSender with D
 
     }
 
+    // retry a few times as the temporaryServerAddress can be taken by the time the new actor system
+    // binds
+    def selectionAndBind(
+        config: Config,
+        thisSystem: ActorSystem,
+        probe: TestProbe,
+        retries: Int = 3): (ActorSystem, ActorSelection) = {
+      val otherAddress = temporaryServerAddress()
+      val otherConfig = ConfigFactory.parseString(s"""
+              akka.remote.netty.tcp.port = ${otherAddress.getPort}
+              """).withFallback(config)
+      val otherSelection =
+        thisSystem.actorSelection(s"akka.tcp://other-system@localhost:${otherAddress.getPort}/user/echo")
```

Comment:
prepare for Artery TCP or is this a classic only test?

Reply:
there are a lot of classic specific things in this test and was made a classic only test under the artery default PR. Some of the tests look to be transport agnostic tho

Reply:
Actually, the transport agnostic tests do look to have an equivalent in `artey.RemoteConnectionSpec`